src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-manager.js

```diff
@@ -109,6 +110,9 @@ define([
                 promise.resolve(media.toJSON());
             } else {
                 media = Media.findOrCreate({id: mediaId});
+                media.clear();
+                media.set({id: mediaId});
```

Comment:
When reading these few lines... Why is the `findOrCreate` method called at all? We always fetch the data again anyway, so this seems wrong...